diff --git a/modules/consensus/esbocs/dap_chain_cs_esbocs.c b/modules/consensus/esbocs/dap_chain_cs_esbocs.c
index 038c57621..6fe9c3907 100644
--- a/modules/consensus/esbocs/dap_chain_cs_esbocs.c
+++ b/modules/consensus/esbocs/dap_chain_cs_esbocs.c
@@ -420,7 +420,7 @@ static void s_check_db_collect_callback(dap_global_db_instance_t UNUSED_ARG *a_d
     size_t l_objs_count = 0;
     pthread_rwlock_wrlock(&s_collecting_lock);
     dap_global_db_obj_t *l_objs = dap_global_db_get_all_sync(a_group, &l_objs_count);
-    if (l_objs_count >= 10) {
+    if (l_objs_count > 0) { // syncfix3-fix4: lower autocollect threshold
         for (size_t i = 0; i < l_objs_count; i++) {
             if (l_objs[i].value_len != sizeof(uint256_t)) {
                 log_it(L_ERROR, "Autocollect record %s is corrupt, len %zu clear it", l_objs[i].key, l_objs[i].value_len);
