diff --git a/modules/consensus/esbocs/dap_chain_cs_esbocs.c b/modules/consensus/esbocs/dap_chain_cs_esbocs.c
index 038c57621..70157d37b 100644
--- a/modules/consensus/esbocs/dap_chain_cs_esbocs.c
+++ b/modules/consensus/esbocs/dap_chain_cs_esbocs.c
@@ -2308,7 +2308,7 @@ static bool s_stream_ch_packet_in(dap_stream_ch_t *a_ch, void *a_arg)
     }
     *l_args = (struct esbocs_msg_args){ a_ch->stream->node, l_session, l_message_size };
     memcpy(l_args->message, l_message, l_message_size);
-    dap_proc_thread_callback_add(l_session->proc_thread, s_process_incoming_message, l_args);
+    dap_proc_thread_callback_add_pri(l_session->proc_thread, s_process_incoming_message, l_args, DAP_QUEUE_MSG_PRIORITY_HIGH); // syncfix3-fix5: prioritize consensus messages
     return true;
 }
 
